templates/yarn-lock-upgrade/template.yaml

```
@@ -60,18 +60,21 @@ steps:
       #if there is a PR already open, close it
       if [[ -n "$PULL_REQUEST_NUMBER" ]] 
       then
-        echo "A PR #$PULL_REQUEST_NUMBER already exists. Closing it."
-        hub issue update $PULL_REQUEST_NUMBER -s closed   
-      fi
-
-      #It should be 'master', but right now it has no files in it
-      if [ "${{parameters.isTest}}" == "False" ]; 
-      then
-        git checkout ${{ parameters.baseBranch }}
+        CURRENT_BRANCH=$(hub pr show "$PULL_REQUEST_NUMBER" -f %H)
+        echo "A PR #$PULL_REQUEST_NUMBER already exists."
+        CREATE_NEW_PR=false
+
+        git checkout $CURRENT_BRANCH
+        git merge origin/${{ parameters.baseBranch }}
```

This instruction is critical as it can fail due to conflicts. Maybe we can mitigate by using an explicit [merge strategy](https://git-scm.com/docs/git-merge#Documentation/git-merge.txt--Xltoptiongt):
```suggestion
        git checkout $CURRENT_BRANCH
        git merge -X theirs origin/${{ parameters.baseBranch }}
```
Once conflicts are solved, there's no need to `push --force`, as the history will be linear.

Another option is to _override_ the current branch and start over from master
```suggestion
        git checkout master
        git branch -D $CURRENT_BRANCH
        git checkout -b $CURRENT_BRANCH
```
This will ensure a clean slate for our changes, but it will require to `push --force` later.